hax/hax/util.py

```diff
@@ -820,6 +835,16 @@ def get_node_ctrl_fids(self,
             list_fids.append(Fid.parse(ctrl_fid))
         return list_fids
 
+    def get_node_hare_motr_s3_fids(self, node: str) -> List[Fid]:
+        services = ['hax', 'ios', 'confd', 's3service']
+        node_data: List[Dict[str, Any]] = self.get_node_health_details(node)
+        fids: List[Fid] = []
+        if node_data:
+            for item in node_data:
+                if item['ServiceName'] in services:
+                    fids.append(mk_fid(ObjT.PROCESS, int(item['ServiceID'])))
```

Comment:
[Optional] This version is more readable:
```suggestion
        if not node_data:
            return []
 
        for item in node_data:
            if item['ServiceName'] in services:
                fids.append(mk_fid(ObjT.PROCESS, int(item['ServiceID'])))

```

Reply:
Okay, done.